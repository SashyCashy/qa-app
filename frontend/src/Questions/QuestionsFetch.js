import React from 'react';
import axios from "axios";


    static getJSONAsync(){
    return fetch('http://localhost:8081/')
        .then(response => response.json())
        .then(data => this.setState({ data }));

    }



export default getJSONAsync;