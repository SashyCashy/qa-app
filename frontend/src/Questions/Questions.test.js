import React from 'react';
import renderer from 'react-test-renderer';
import QuestionsFetch from './QuestionsFetch';
import Questions from './Questions';
import axios from 'axios';


test('Link changes the class when hovered', () => {

    const wrapper = renderer.create(<Questions />);
    let tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();

 });





/*test('should fetch users', () => {
  const resp = {data: [{name: 'Bob'}]};
  axios.get.mockResolvedValue(resp);

  // or you could use the following depending on your use case:
  // axios.get.mockImplementation(() => Promise.resolve(resp))

  return Users.all().then(users => expect(users).toEqual(resp.data));
});*/