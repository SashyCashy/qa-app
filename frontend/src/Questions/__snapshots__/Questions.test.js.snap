// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <p>
       Loading Questions... 
    </p>
  </div>
</div>
`;
